Dockerfile

```diff
@@ -18,7 +18,7 @@ RUN if [ "$TARGETARCH" = "arm64" ] ; then \
   fi
 
 # STEP 2: Build small image
-FROM --platform=${BUILDPLATFORM} registry.k8s.io/build-image/distroless-iptables:v0.6.6
+FROM --platform=${TARGETPLATFORM} registry.k8s.io/build-image/distroless-iptables:v0.6.6
```

Comment:
Not familiar with cross building, Do we need to pase TARGETPLATFORM in an ARG statement?